packages/rolldown/src/cli/index.ts

```diff
@@ -10,7 +10,7 @@ async function main() {
 
   if (cliOptions.config) {
     await bundleWithConfig(cliOptions.config, cliOptions)
-    process.exit(0)
```

Comment:
I think we could also remove the rest of `process.exit`?

Reply:
done